compiler/runtime/OMRRuntimeAssumptions.hpp

```diff
@@ -105,10 +105,13 @@ class RuntimeAssumption : public TR_Link0<RuntimeAssumption>
       _nextAssumptionForSameJittedBody = 0;
       setNext(NULL);
       }
```

Comment:
I'd like to see Doxygen comments for new functions added to the API please.

Reply:
Might as well also properly Doxygen `_nextAssumptionForSameJittedBody` since we are changing its meaning.